CMake/Modules/FindPhysicsFS.cmake

```diff
@@ -0,0 +1,59 @@
+#.rst:
```

Comment:
I modified the original that comes with CMake but i never tried to use the PHYSFSDIR environment variable. Thanks.